example/datasets/svhn/dataset.py

```diff
@@ -0,0 +1,62 @@
+import io
+
+import h5py
```

Comment:
Add requirements.txt or Starwhale Runtime to describe h5py dependency?

Reply:
done